test/initializer_test.rb

```diff
@@ -12,24 +12,50 @@ class ArgumentativeStorer
     initialize_with :foo, :bar
   end
 
+  class KeywordStorer
+    include Mandate
+    initialize_with :foo, bar: ["default"]
```

Comment:
Would you mind changing this from `["default"]` to "default" just to show the array isn't some special syntax pls?

Reply:
That's a fair point -- implemented.